colossus-docs/src/main/paradox/services.md

```diff
@@ -81,3 +85,13 @@ To configure via code, create a `ServiceConfig` object and pass it to the `Reque
 @@snip [ServiceConfigExample.conf](../scala/ServiceConfigExample.scala) { #example }
 
 @@snip [ServiceConfigExample.conf](../scala/ServiceConfigExample.scala) { #example1 }
+
+`RequestHandler` allows you to configure how request errors are reported. The `ServiceConfig` case class contains the
```

Comment:
This description is very implementation specific. For a user of colossus, they just need to know that they can suppress errors by adding the config, or override the whole thing by providing their own implementation of `RequestFormatter` in code.